resources/views/pdf/bookingContract.blade.php

```diff
@@ -8,7 +8,7 @@ class="max-w-[200px] max-h-[100px] mx-auto" />
 </x-pdf.contractHeader>
 <x-pdf.section>
     <strong>{{ $booking->band->name }}</strong> (hereinafter referred to as "Artist"), enter into this Agreement
-    with <strong>{{ $booking->contacts[0]->name }}</strong> (hereinafter referred to as "Buyer"), for the engagement of a live musical performance
```

Comment:
@qskousen well, that will do it. I was specifically adding the signer contact to the view and doing absolutely nothing with it.